projects/demo/src/modules/components/textfield/index.html

```diff
@@ -20,5 +20,69 @@
         />
     </ng-template>
 
-    <tui-setup *pageTab="'Setup'" />
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-textfield
+                [iconEnd]="icons.iconEnd"
```

Comment:
We also need `filler` here. Let's not show other missing inputs for now (content and stringify)

Reply:
Done